internals/notifier.py

```diff
@@ -840,21 +840,22 @@ class OTExtensionApprovedHandler(basehandlers.FlaskHandler):
   def process_post_data(self, **kwargs):
     self.require_task_header()
     feature = self.get_param('feature')
-    if feature is None:
-      self.abort(400, 'No feature provided.')
     gate_id = self.get_param('gate_id')
-    if gate_id is None:
-      self.abort(400, 'Extension gate ID not provided.')
     requester_email = self.get_param('requester_email')
-    if not requester_email:
-      self.abort(400, 'Extension requester\'s email address not provided.')
+    ot_display_name = self.get_param('ot_display_name')
```

Comment:
This will fail for any tasks currently in the queue that lack `ot_display_name`, but that should be OK because those tasks are short-lived and happen at a different time of day than we usually do deployments.

Reply:
This should not be a problem, since the `ot_display_name` field is required to be populated on the OT creation form 🙂